website/docs/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+id: installation
+title: Installation
+---
+
+:::note
+You need to run the following steps on a dokku server.
+If you do not have a dokku server for now, please follow the getting started guide.
+:::
+
+First we need to create a new app for ledokku.
+
+```sh
+dokku apps:create ledokku
+```
+
+Create a new Redis database.
+
+```sh
+# If the Redis plugin is not installed on your server run the following command
+sudo dokku plugin:install https://github.com/dokku/dokku-redis.git redis
+
+dokku redis:create ledokku
+dokku redis:link ledokku ledokku
+```
+
+Create a new Postgres database.
+
+```sh
+# If the Postgres plugin is not installed on your server run, otherwise skip this command
+sudo dokku plugin:install https://github.com/dokku/dokku-postgres.git postgres
+
+dokku postgres:create ledokku
+dokku postgres:link ledokku ledokku
+```
+
+In your browser open https://github.com/settings/developers and click on the "New OAuth App" button.
+
+Add a name, a homepage url, and in the field "Authorization callback URL" set the value to your ledokku ip address and port 4000. (eg: http://123.123.123.123:4000)
```

Comment:
@pradel maybe here we can add example of homepage url as well? As for someone setting everything up for the first time it could not be obvious.